edit-post/api/README.md

```diff
@@ -3,35 +3,57 @@ Edit post API
 
 The edit post API contains the following methods:
 
-### `wp.editPost.__experimentalRegisterSidebar( name: string, settings: { title: string, render: function } )`
+### `wp.editPost.__experimental.registerPlugin( { name: string, render: function } )`
```

Comment:
This is a huge limitation of our docs that there is no easy way to keep some parts in sync. It would be nice to come up with a solution. Maybe we should keep some docs next to the code but also expose them as part of the handbook?